### PR TITLE
ghostscript: update to version 9.52

### PR DESCRIPTION
**Pull request by nickgaya**
#### Description

* update to version 9.52
* add patch to build gpdl with shared libjpeg (see https://bugs.ghostscript.com/show_bug.cgi?id=702474)
* clean up unused code

Closes: https://trac.macports.org/ticket/60540

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.13.6 17G13035
Xcode 10.1 10B61

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] <strike>tried existing tests with `sudo port test`?</strike> (not applicable)
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: print/ghostscript/Portfile [hunk 1]
```diff
@@ -32,12 +32,13 @@ distfiles           ${distname}.tar.gz:source \
                     ${mappingresources_commit}.zip:misc
 
 patchfiles          patch-base_unix-dll.mak.diff \
```
Comment (cjones051073) [inline print/ghostscript/Portfile:34]: If this patchfile is no longer needed please remove it aswell.

**Comment by cjones051073** [comment]
There are merge conflicts that need adressing

File: print/ghostscript/Portfile (hunk 1 above)
Comment (nickgaya) [reply print/ghostscript/Portfile:34]: The makefile patches are still needed.

**Comment by ryandesign** [comment]
This update was done separately in bbb7d5d71d3fbf78ab292a093579102c3686bb24 and 8b0b4b7aa175fff67eae8c6258b3b6254a9e437c.

**Comment by nickgaya** [comment]
Hi @ryandesign and @drkp, I'm a bit confused why this PR sat inactive for a month with no input and then someone else made the same changes directly on master (minus some of the cleanup of unused config in this PR).

What is the recommended way to contribute to MacPorts, if GitHub PRs are not used?

**Comment by ryandesign** [comment]
Sorry about that. Not every developer remembers to check for open PRs before updating a port themselves. Please do continue to contribute via PRs; I've been merging several every day and other developers merge them too.

**Comment by drkp** [comment]
Ah, that was my mistake. I saw the ticket (thanks for tracking down the patch to backport!) when I was going through and updating various TeX dependencies, but I didn't see that there was an associated PR. Sorry for not checking here too.

(It'd be helpful if we had a way to search for both open tickets and open PRs together, but I'm not sure that's possible with one here and one on trac...)

**Comment by ryandesign** [comment]
Another thing you can do: If your PR solves a ticket, add a comment to the ticket with the PR URL. It might be nice if this happened automatically, but for now it doesn't.

**Comment by nickgaya** [comment]
Thanks for the info, in any future PRs I'll add a link from Trac back to GitHub if there's an open ticket.

**Pull request closed without merging**